### ISSUE TITLE
Energies in cycle_.trj

### ISSUE THREAD

**Issue by O2-AC** [eljost/pysisyphus#190]
I just updated from a rather old version (0.4rc8) to the newest version. About everything works like expected, however, when I perform neb calculations and try to view the last `cycle_xx.trj` with `molden` and open the graph for Energy vs step, I get a flat line. 
Running `pysisplot --cosens` produces the correct plots and also the correct `splined_hei.xyz` is written.

When I compare the output to a file with the old version I see that for the first structure the entry differs on the 2nd line:

old:
```
10
-13.5317100475 ,  energy: -13.531710047663 gnorm: 0.000390694101 xtb: 6.4.0 (d4b70c2)
  C -1.20173307  0.05070977 -0.48763606
  C  0.30806079  0.05520405 -0.53071241
  C  1.02544775 -0.08229238  0.79133274
  O  0.91070273  0.16315904 -1.56891950
  H -1.59293143 -0.05390150  0.51834698
  H -1.56847468 -0.76446412 -1.10911718
  H -1.56848725  0.97630635 -0.92810146
  H  0.34514081 -0.16964117  1.63129365
  H  1.67114252  0.78284430  0.93227327
  H  1.67113184 -0.95792434  0.75123997
```

new:
```
10
        -13.53171005,
  C -1.20173307  0.05070977 -0.48763606
  C  0.30806079  0.05520405 -0.53071241
  C  1.02544774 -0.08229238  0.79133274
  O  0.91070273  0.16315904 -1.56891950
  H -1.59293143 -0.05390150  0.51834698
  H -1.56847468 -0.76446412 -1.10911718
  H -1.56848725  0.97630635 -0.92810146
  H  0.34514081 -0.16964117  1.63129365
  H  1.67114252  0.78284430  0.93227327
  H  1.67113184 -0.95792434  0.75123996
```

I'm using the following `.yaml` for the new version:
```
preopt:
  max_cycles: 15
cos:
  type: neb          
  climb: true
opt:
  type: lbfgs           
  max_cycles: 50   
  align: True       
interpol:
  type: idpp
  between: 15       
calc:
  type: xtb          
  charge: 0
  mult: 1
  pal: 4
geom:
  fn: [start.xyz, end.xyz]
```
And this `.yaml` for the old version:
```
cos:
  type: neb      
  climb: true
opt:
  type: qm         
  max_cycles: 10   
  align: True      
  climb: True       
interpol:
  type: lst
  between: 5      
calc:
  type: xtb        
  charge: 0
  mult: 1
  pal: 4
xyz: [start.xyz, end.xyz]
```

Thanks in advance.

**Comment by O2-AC** [comment]
You can scratch my comment on the difference of the output file, I just figured out that the additional information in the old file comes from an xTB optimisation of the starting structure which I've done manually when I used the older version.

But by some troubleshooting I found, that in the old version there was a space between the energie value and the separating comma, whereas this space is missing in the output file when using the newer version of pysisyphus.
By manually adding that space, `molden` recognizes the energies and is able to show the proper plot. So if this change in format doesn't break other functionality, compatibility with `molden` can be restored by explicitly adding that space when writing the `.trj` files.

**Comment by eljost** [comment]
Thanks for the hint. I'll see if it can be fixed.

**Comment by eljost** [comment]
This will be fixed in the final 0.7.5 release. Thanks for reporting

**Comment by O2-AC** [comment]
Great to hear! Looking forward to the release.
